traffic_portal/app/src/common/modules/form/deliveryService/form.deliveryService.DNS.tpl.html

```diff
@@ -343,6 +342,34 @@ <h3 ng-if="!open()">Previous Value</h3>
                             </div>
                         </div>
                     </div>
+                    <div class="form-group" ng-class="{'has-error': hasError(generalConfig.requiredCapabilities), 'has-feedback': hasError(generalConfig.requiredCapabilities)}">
+                        <label class="has-tooltip control-label col-md-2 col-sm-2 col-xs-12" for="capability">Required Capability
+                            <div class="helptooltip">
+                                <div class="helptext">Required capability(ies) associated with a delivery  service</div>
+                            </div>
+                        </label>
+                        <div class="col-md-10 col-sm-10 col-xs-12">
+                            <div class="checkbox-scroll-container">
+                                <div ng-repeat="capability in requiredCapabilities">
+                                    <input
+                                            type="checkbox"
+                                            id="capability"
+                                            name="capability"
+                                            ng-model="selectedCapabilities[capability.name]"
+                                            ng-checked="selectedCapabilities[capability.name]"
+                                    />
+                                    <label>
+                                        <a ng-href="/#!/server-capabilities/edit?name={{capability.name}}" target="_blank"> {{capability.name}}</a>
+                                    </label>
```

Comment:
From [the MDN documentation on the label element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label):

> Don't place interactive elements such as [anchors](https://github.com/apache/trafficcontrol/pull/7314/a.html) or [buttons](https://github.com/apache/trafficcontrol/pull/7314/button.html) inside a label. Doing so makes it difficult for people to activate the form input associated with the label.

By design, a label is associated with a form control. Clicking on the label is intended to be identical to clicking on the form control - but that can't be the case if it has a different behavior on click. Furthermore, in order to be associated with an input, every label must either contain the element it labels or have a [`for` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attr-for) set to the ID of the element it labels.

Reply:
Moving `.has-tooltip` of of the label breaks the dotted underline. So the underline should be re-added to the label somehow.

https://github.com/apache/trafficcontrol/blob/dad57721e2a82ad0fb5eb1d30ba2ca33542d68ec/traffic_portal/app/src/common/modules/form/_form.scss#L116-L117

Reply:
Fix for dotted underline: https://github.com/apache/trafficcontrol/pull/7348